examples/mongodb/primary.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"gopkg.in/mgo.v2"
+	"log"
+)
+
+var url = flag.String("url", "localhost:27017", "url of the mongoserver")
```

Comment:
I very much like this - the code is identical in every single way to a file I had sitting in a branch I'd forgotten about. Very pleased to see it here.